lib/src/IndexingPolicy.cpp

```diff
@@ -37,15 +37,17 @@ using namespace web::json;
 IndexingPolicy::IndexingPolicy()
 	: automatic_(true)
 	, indexing_mode_(IndexingMode::CONSISTENT)
-	, excluded_paths_(0)
+	, excluded_paths_()
 	, included_paths_(0)
 {
+	included_paths_.push_back(make_shared<Index>(IndexType::HASH, 3, 3, _XPLATSTR("/")));
+	included_paths_.push_back(make_shared<Index>(IndexType::RANGE, 6, 3, _XPLATSTR("/\"_ts\"/?")));
 }
 
 IndexingPolicy::IndexingPolicy(
 	const bool automatic,
 	const IndexingMode& indexing_mode,
-	const vector<shared_ptr<IndexPath>>& included_paths,
+	const vector<shared_ptr<Index>>& included_paths,

```

Comment:
are you really sure about this? [This](https://docs.microsoft.com/en-us/rest/api/cosmos-db/collections) tells me it is not proper way?

Reply:
This is the same as above. It seems that I override the original source that I received and ignore the changed contents in the middle.